charts/prometheus-multipathd-exporter/Chart.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v2
+name: prometheus-multipathd-exporter
+description: A Helm chart for multipathd_exporter
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
FYI, release-please will auto bump this automatically moving forward once you configure it :)